ia-wishlist-bot/get_wishlist_editions_isbn.py

```diff
@@ -0,0 +1,38 @@
+"""
+Python Script to get a list of ISBNs which aren't added to Open Library
+
+Input: 'wishlist_works_edition.ndjson'
+* Contains the parameters "oclc_synonyms",  "openlibrary_synonyms", "editions" -> {"isbn", "olid"}
+
+Output: 'ol_works.csv'
+* Contains the Parameter: 'isbn-13' for all the books which aren't added to Open Library
+"""
+
+import ndjson
+import csv
+import time
+
+new_data = []
+
+
+with open('/storage/openlibrary/wishlist/wishlist_works_editions.ndjson') as f:
```

Comment:
We may want to point this to an archive.org item (rather than this relative path to a file which cannot be accessed by future readers of this script)